diff -dur -x '*.orig' astroid-1.4.1.orig/astroid/manager.py astroid-1.4.1/astroid/manager.py
--- astroid-1.4.1.orig/astroid/manager.py	2015-11-29 21:37:16.000000000 +0100
+++ astroid-1.4.1/astroid/manager.py	2015-12-07 10:27:07.000000000 +0100
@@ -118,7 +118,7 @@
                 module = self.zip_import_data(filepath)
                 if module is not None:
                     return module
-            elif mp_type in (imp.C_BUILTIN, imp.C_EXTENSION):
+            elif mp_type in (imp.C_BUILTIN, imp.C_EXTENSION, imp.PY_COMPILED):
                 if mp_type == imp.C_EXTENSION and not self._can_load_extension(modname):
                     return self._build_stub_module(modname)
                 try:
@@ -127,9 +127,6 @@
                     msg = 'Unable to load module %s (%s)' % (modname, ex)
                     raise exceptions.AstroidBuildingException(msg)
                 return self.ast_from_module(module, modname)
-            elif mp_type == imp.PY_COMPILED:
-                msg = "Unable to load compiled module %s" % (modname,)
-                raise exceptions.AstroidBuildingException(msg)
             if filepath is None:
                 msg = "Unable to load module %s" % (modname,)
                 raise exceptions.AstroidBuildingException(msg)
diff -dur -x '*.orig' astroid-1.4.1.orig/astroid/modutils.py astroid-1.4.1/astroid/modutils.py
--- astroid-1.4.1.orig/astroid/modutils.py	2015-11-29 21:37:16.000000000 +0100
+++ astroid-1.4.1/astroid/modutils.py	2015-12-07 10:26:47.000000000 +0100
@@ -563,6 +563,10 @@
         return None, imp.C_BUILTIN
     elif mtype == imp.PKG_DIRECTORY:
         mp_filename = _has_init(mp_filename)
+        try:
+            return get_source_file(mp_filename), imp.PY_SOURCE
+        except NoSourceFile:
+            return mp_filename, imp.PY_COMPILED
         mtype = imp.PY_SOURCE
     return mp_filename, mtype
 
