--- astroid-2.3.3/astroid/__pkginfo__.py.orig	2019-11-16 17:17:15.331282360 +0100
+++ astroid-2.3.3/astroid/__pkginfo__.py	2019-11-16 17:24:15.249007470 +0100
@@ -22,7 +22,7 @@
 install_requires = [
     "lazy_object_proxy>=1.4.0",
     "wrapt>=1.11,<1.13",
-    'typed-ast>=1.4.0,<1.5;implementation_name== "cpython" and python_version<"3.8"',
+    "typed-ast>=1.4.0,<1.5",
 ]
 
 # pylint: disable=redefined-builtin; why license is a builtin anyway?
